gridsome/lib/webpack/createBaseConfig.js

```diff
@@ -126,14 +126,12 @@ module.exports = (app, { isProd, isServer }) => {
 
   // css
 
-  createCSSRule(config, 'css', /\.css$/)
-  createCSSRule(config, 'postcss', /\.p(ost)?css$/)
-  createCSSRule(config, 'scss', /\.scss$/, 'sass-loader', projectConfig.scss)
-  createCSSRule(config, 'sass', /\.sass$/, 'sass-loader', Object.assign({ indentedSyntax: true }, projectConfig.sass))
-  createCSSRule(config, 'less', /\.less$/, 'less-loader', projectConfig.less)
-  createCSSRule(config, 'stylus', /\.styl(us)?$/, 'stylus-loader', Object.assign({
-    preferPathResolver: 'webpack'
-  }, projectConfig.stylus))
+  createCSSRule(config, 'css', /\.css$/, null, projectConfig.css.loaderConfig.css)
```

Comment:
I've got error on this line - cannot read `css` of undefined when used [this config](https://github.com/gridsome/gridsome/pull/46#issuecomment-450556456). Seems like there is some confusion between `loaderConfig` vs `loaderOptions` names.

Reply:
I'll take a look!